content/en/tracing/other_telemetry/connect_logs_and_traces/php.md

```diff
@@ -24,8 +24,95 @@ further_reading:
 
 ## Automatic injection
 
-Given the many different ways to implement logging in PHP<span class="x x-first x-last">,</span> with some completely circumventing PHP's built-in error-logging API, the Datadog PHP tracing library cannot reliably inject trace and span <span class="x x-first x-last">IDs</span> into logs automatically.
-See the section below to learn how to connect your PHP Logs and traces manually.
+Starting in version **0.89.0**, the PHP tracer automatically injects trace correlation identifiers into application logs. For automatic injection to be enabled, the environment variable (resp. directive) `DD_TRACE_LOGS_ENABLED` (resp. `datadog.trace.logs_enabled`) must be set to `true`.
+
+The PHP tracer supports PSR-3 compliant loggers, such as [Monolog][4] or [Laminas Log][5].
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> It is strongly encouraged setting up your logging library to produce your logs in JSON format to:
+  <ul>
+    <li>Avoid the need for <a href="/logs/log_configuration/parsing">custom parsing rules</a>.</li>
+    <li>Ensure that stack traces are properly wrapped into the log event.</li>
+  </ul>
```

Comment:
```suggestion
  <strong>Note</strong>: Set up your logging library to produce logs in JSON format so that:
  <ul>
    <li>You don't need <a href="/logs/log_configuration/parsing">custom parsing rules</a>.</li>
    <li>Stack traces are properly wrapped into the log event.</li>
  </ul>
```